### DISCUSSION TITLE
Some basic questions

### DISCUSSION THREAD

**Discussion by supertick** [Pi4J/pi4j#279] [General]
Hello !
I've been using pi4j v1 for a while, but failed to get gpio pins to work correctly, we did manage to get i2c working fine.
I was looking for a maven dependency list for pi4j v2 which includes gpio & i2c support.
Also a bit confused about versions looks like 2.0 is available in maven central but there are newer releases 2.3 in github?
What would be the repo url to add to the pom ?
Using 2.0 my code lights up like a red Christmas tree:
I2CBus & I2CDevice, GpioPinDigitalMultipurpose all cannot resolve to a type .. :cry: 
Help !?

**Comment by FDelporte** [comment]
Hi, V2 is indeed totally different compared to V1, please take a look at https://pi4j.com/getting-started/minimal-example-application/ for a first example.

Latest release: V2.3.0 (2023-02-06, see [Release Notes](https://pi4j.com/about/release-notes/)).
